services/catalog/tests/unit/with_dbs/conftest.py

```diff
@@ -41,7 +42,7 @@ def app(
 
 
 @pytest.fixture
-def client(app: FastAPI) -> Iterable[TestClient]:
+def client(loop: asyncio.AbstractEventLoop, app: FastAPI) -> Iterable[TestClient]:
```

Comment:
so this will provide a loop to all downstream tests, right? I think it is a good "common denominator" fixture since most of the async tests use it. Now, what i find strange is why the tests downstream would need to be async since you are returning a synchronous client to query the app ...

Reply:
so the TestClient of starlette while being synchronous, needs a running loop